### PR TITLE
Advance parts

### PR DESCRIPTION
**Pull request by eggrobin**
Another attempt at putting those calls in the right order.

### CODE REVIEW

File: ksp_plugin/plugin.cpp [hunk 1]
```diff
@@ -585,9 +585,7 @@ void Plugin::SetPartApparentDegreesOfFreedom(
       part, world_to_apparent_bubble(degrees_of_freedom));
 }
 
-void Plugin::AdvanceTime(Instant const& t, Angle const& planetarium_rotation) {
-  VLOG(1) << __FUNCTION__ << '\n'
```
Comment (pleroy) [inline ksp_plugin/plugin.cpp]: May be keep the VLOG for the time?

File: serialization/journal.proto [hunk 2]
```diff
@@ -100,6 +100,17 @@ message AdvanceTime {
...
+    optional AdvanceParts extension = 5120;
+  }
+  message In {
+    required fixed64 plugin = 1 [(pointer_to) = "Plugin", (is_subject) = true];
+    required double t = 2;
+  }
+  optional In in = 1;
+}
+
 message PrepareToReportCollisions {
```
Comment (pleroy) [inline serialization/journal.proto]: This message doesn't seem to be in alphabetical order.

File: ksp_plugin/plugin.cpp (hunk 1 above)
Comment (eggrobin) [inline ksp_plugin/plugin.cpp]: We never actually use the VLOGs, we either journal or add ad hoc traces.

File: serialization/journal.proto (hunk 2 above)
Comment (eggrobin) [inline serialization/journal.proto]: Done.

**Pull request merged**